merlin/models/tf/inputs/embedding.py

```diff
@@ -190,6 +190,7 @@ def __init__(
                 **table_kwargs,
             )
         self.combiner = combiner
+        self.supports_masking = True
```

Comment:
It seems that `supports_masking` is currently only used by `layer_test()`, as masking was removed before.
Not sure if the new solution for masking (when we add session/sequential based support will use `support_masking`

Reply:
According to the keras docs, the Embedding layer supports masking, so I put this in to maintain the functionality of the `Embedding` layer. However, we don't have any tests that use the masking with this `EmbeddingTable` wrapper.